charter/index.md

```diff
@@ -0,0 +1,211 @@
+---
+title: Font and Text Community Group Charter
+---
+This charter:  {{ page.url | remove: "index.html" | prepend: site.url }}
+
+Feedback about this document:
+    https://github.com/font-text/font-text.github.io/issues
+
+## Scope
+
+The Font and Text community group gathers individuals and organisations
+interested in discussing and developing specifications and implementations
+for technologies such as shaping and layout
+which operate on and at the interface between
+text encoding and font formats.
+
+Under the present charter, the Community Group will
+determine a structure and process
+under which such specifications may be created
+by which the greatest number of interested parties may be able to participate.
+Until such a structure and process is determined,
+no technical development of specifications or implementations
+will be undertaken by this Community Group
+and no technical contributions will be accepted.
+
+## Deliverables
+
+* This Community Group aims to produce a proposed Charter under which
+this community should conduct its technical work,
+either to be used as an updated Charter for this CG,
+or to set up a different group,
+possibly a W3C Working Group,
+to conduct that work.
+
+* If the Community Groups determines that it would be appropriate
+to divide the technical work into several venues,
+it may produce several such proposed charters
+rather than a single one.
+
+* Various supporting documents and articles
+describing the problem space
+may also be produced.
+
+All deliverables of this Community Group will use the
+[W3C Software and Document License](http://www.w3.org/Consortium/Legal/2015/copyright-software-and-document).
+
+This group maintains an [index](https://font-text.github.io/work-items/)
+of the documents it is working on.
+
+## Communication
+
+This group operates under the
+[W3C Code of Ethics and Professional Conduct ](https://www.w3.org/Consortium/cepc/).
+
+This group will conduct its work publicly,
+primarily using the public-font-text@w3.org mailing list
+([archives](https://lists.w3.org/Archives/Public/public-font-text/)
+for minutes, announcements, and high level discussions,
+and [GitHub](https://github.com/font-text)
+for detailed discussions
+(See [Contribution Mechanics](#contribution-mechanics).
+
+Teleconferences and face-to-face meetings
+may be held from time to time,
+provided sufficient advance notice is given.
+Minutes of any such meeting will be
```

Comment:
Might be good to clarify what may constitute "minutes". 

Thus far, the informal meetings have had transcripts, which are not the same thing as minutes. Transcripts can be helpful to capture the thought process that led to a result. But two shortcomings are that they are inevitably an incomplete record (wrt statements made, tone and general sentiment...), and knowledge that there will be published transcripts can stifle free discussion.

Minutes, typically, are a record of outcomes—decisions and action items.

Reply:
Agree. I will clarify.

Reply:
In w3c lingo, "minutes" refers to a combination of the two: a transcript, made as accurate as can be based on the typing abilities of the person assigned / volunteering for this task and the corrections provided by other participants, and within this (and preferably also repeated as a summary at the start or end), explicit decisions and actions are called out.

Formatting my vary, but here's an example of what I'd consider as good minutes: https://lists.w3.org/Archives/Public/www-style/2020May/0022.html

W3C also has an IRC bot to help you generate that sort of things:
* sample output: https://w3c.github.io/scribe2/sample-member.html
* bot documentation: https://w3c.github.io/scribe2/scribedoc.html

To me, the w3c context implies this, and this is reinforced by "and any resolution will be called out explicitly" a couple of lines below. But if it isn't clear enough, we should certainly rephrase to make it clear.

Reply:
In addition to the WG example @frivoal provides, here are some CG examples for your reference:

https://github.com/w3c/miniapp/blob/gh-pages/Meetings/Meetings.md

Reply:
I think this request for clarification has been addressed, both in comments here but also in the text of the charter itself. I'm marking this as resolved but if there is any ongoing concern here please do ping again and we can review.